helm/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: flink-operator
+description: A Helm chart for Flink operator
+version: 0.8.2
+appVersion: 0.4.0
```

Comment:
Thoughts on having the chart config and templates against master version `0.5.0` ?  Ideal to keep it in sync to avoid customer surprises.